src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int side;
+
+    public Square(int side, String color) {
+        super(color);
+        this.side = side;
+    }
+
+    @Override
+    public double getArea() {
+        return side * side;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: square, area: " + String.format("%.1f", getArea())
```

Comment:
Use 'name()' instead of 'toString()' when you want to get the String representation of enum constants. If 'getColor()' is an enum, you should ensure it uses 'name()' internally.